### PR TITLE
testingmerge

### PR DESCRIPTION
**Pull request by RussleKelly**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -16,4 +16,6 @@ enerbipornbiopnebpore
 one more change in separate branch
 hello there
 
-one more change
\ No newline at end of file
+one more change
```
Comment (RussleKe) [inline README:19]: This is weird change

Comment (RussleKelly) [reply README:19]: nah, it's fine

**Comment by RussleKelly** [comment]
Test comment from offline

**Comment by RussleKelly** [comment]
Test from offline

**Comment by RussleKelly** [comment]
Onre more test comment

File: README [hunk 2]
```diff
@@ -9,6 +9,9 @@ Author : Scott Chacon
 dsgneronvboenoie
```
Comment (RussleKelly) [inline README]: That's quite weird

File: README [hunk 3]
```diff
@@ -3,4 +3,22 @@ SimpleGit Ruby Library
```
Comment (RussleKelly) [inline README:3]: look here

File: README [hunk 4]
```diff
@@ -14,4 +14,6 @@ bru bru bur bru
 enerbipornbiopnebpore
 enerbipornbiopnebpore
 
-hello there
\ No newline at end of file
+hello there
+
```
Comment (RussleKelly) [inline README:23]: I don't get it

**Pull request merged**